packages/algoliasearch-helper/index.d.ts

```diff
@@ -1457,6 +1463,25 @@ declare namespace algoliasearchHelper {
       exhaustive: boolean;
     }
   }
+
+  export type PlainRecommendParameters =
+    | RecommendationsQuery
+    | TrendingQuery
+    | RecommendedForYouQuery;
+
+  export type PlainRecommendParametersWithId = PlainRecommendParameters & {
+    $$id: string;
+  };
```

Comment:
I added a specific type for recommend parameters augmented with the internal identifier, so we can rely on `PlainRecommendParameters` for the queries actually sent to the API.

While not specified in the RFC, I chose here `$$id` as the name for the id key, but I'm open to anything else.